easybuild/easyconfigs/m/METIS/METIS-5.1.0-foss-2017b.eb

```diff
@@ -0,0 +1,26 @@
+name = 'METIS'
+version = '5.1.0'
+
+homepage = 'http://glaros.dtc.umn.edu/gkhome/metis/metis/overview'
+description = """METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes,
+ and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the
+ multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD',
+]
+
+checksums = ['76faebe03f6c963127dbb73c13eab58c9a3faeae48779f049066a21c087c5db2']  # metis-5.1.0.tar.gz
```

Comment:
please include checksum for patch too & reorder (below `patches` line), easiest way is to use `--inject-checksums`